src/app/page.tsx

```diff
@@ -7,13 +7,13 @@ import Features from "@/components/LandingPage/Features";
 import Footer from "@/components/LandingPage/Footer";
 import Working from "@/components/LandingPage/Bug";
 import Bug from "@/components/LandingPage/Bug";
-
+import GoToTop from "@/components/LandingPage/scrollTOtop";
 export default function Component() {
```

Comment:
can we have the file name as scrollToTop ? instead of scrollTOtop !!

Reply:
sure 

> can we have the file name as scrollToTop ? instead of scrollTOtop !!

sure